lib/rsk-utils.js

```diff
@@ -477,7 +480,7 @@ const findBridgeTransactionsInThisBlock = async (web3Client, blockHashOrBlockNum
   // const bridgeTransactionParser = new BridgeTransactionParser(web3Client);
   // return bridgeTransactionParser.getBridgeTransactionsInThisBlock(blockHashOrBlockNumber);
 
-  return bridgeTxParser.getBridgeTransactionsInThisBlock(blockHashOrBlockNumber);
+  return await bridgeTxParser.getBridgeTransactionsInThisBlock(blockHashOrBlockNumber);
```

Comment:
Should we remove the `async` from the definition of the method?

Reply:
Why? We are using `await` now, so we need it.